<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Library</class>
 <widget class="QMainWindow" name="Library">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/inori_zirong_chibi.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblHeader">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>347</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ff0000;&quot;&gt;Multimedia Collection&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>631</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>New Collection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCopy">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>View Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>851</width>
      <height>551</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>849</width>
       <height>549</height>
      </rect>
     </property>
     <widget class="QTableView" name="tbCollection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>421</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/greenBackGround.jpg);</string>
      </property>
      <widget class="QLabel" name="lblCOLLTITLE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblCOLLTYPE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblCOLLDATA">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Data Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblCOLLVERSION">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Version&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblIDENCODE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Iden Code&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnCollTitle">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnCollVersion">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnCollIndenCode">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblIDENCODE_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Publisher&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnCollPublisher">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblIDENCODE_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Year of Record&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblIDENCODE_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Duration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnCollDuration">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>360</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblIDENCODE_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff00ff;&quot;&gt;Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbCollType">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbCollDataType">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbCollCategory">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>410</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAdd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>460</y>
         <width>115</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCancel">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>460</y>
         <width>115</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbYearRecord">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>310</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_image" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QLabel" name="lblImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/221655.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPicture">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
     </property>
     <property name="text">
      <string>Set Picture</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cbType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDataType">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>610</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCategory">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>610</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFilter">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>610</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>lblHeader</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>widget_image</zorder>
   <zorder>cbType</zorder>
   <zorder>cbDataType</zorder>
   <zorder>cbCategory</zorder>
   <zorder>btnFilter</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
